build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.6.4'
```

Comment:
Is it possible to use latest stable 4.1.1? @anangkur

Reply:
I have tried it but it doesn't have gradle.wrapper.properties so I cannot update it. because it also has to update the gradle to 6.5 in gradle.wrapper.properties. can I create the gradle.wrapper.properties manually mas @mochadwi ? I don't really understand about the gradle things actually.

Reply:
Checking the file

Reply:
https://www.github.com/sampingantech/Image-Support/tree/master/gradle%2Fwrapper

I see, this is the problem yah. Hmm okay, stick to the latest 3.y.z for now thanks

Reply:
yup mas, thank you 🙏🏽